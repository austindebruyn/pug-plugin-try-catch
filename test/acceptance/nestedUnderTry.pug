.foo
  try
    try
      h1= missingLocal.name
    catch
      h1 caught!
  catch
