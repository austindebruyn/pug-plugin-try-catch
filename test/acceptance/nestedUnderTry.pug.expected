<div class="foo"><h1>caught!</h1></div>
